spec/services/experimentation/data_export_spec.rb

```diff
@@ -0,0 +1,166 @@
+require "rails_helper"
+
+RSpec.describe Experimentation::DataExport, type: :model do
+
+  TIME_FORMAT = "%Y-%m-%d"
+
+  def create_notification(experiment, template, patient, appt)
+    create(:notification,
+      experiment: experiment,
+      message: template.message,
+      patient: patient,
+      purpose: :experimental_appointment_reminder,
+      remind_on: appt.scheduled_date + template.remind_on_in_days.days,
+      reminder_template: template,
+      status: "sent",
+      subject: appt
+    )
+  end
+
+  describe "#as_csv" do
+    it "exports accurate data in the expected format" do
```

Comment:
These specs are both tedious and incomplete, but they did help me find some bugs.